src/Microsoft.DotNet.DeadRegionAnalysis/AnalysisEngine.Options.cs

```diff
@@ -0,0 +1,200 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.MSBuild;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.DotNet.DeadRegionAnalysis
+{
+    public partial class AnalysisEngine
+    {
+        public class Options
+        {
+            private IEnumerable<IReadOnlyDictionary<string, Tristate>> _symbolConfigurations;
+            private Tristate _undefinedSymbolValue;
+
+            public IEnumerable<Document> Documents { get; private set; }
```

Comment:
Enumerable suggests the values can change.  If they can't we should use `ImmutableArray<Document>` instead.